index.html

```diff
@@ -7,59 +7,61 @@
   <script src="app.js"></script>
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
   <link rel="stylesheet" href="style.css">
+  <link rel="shortcut icon" type="image/x-icon" href="favicon.ico"/>
 </head>
 <body ng-controller="Damage">
-<div class="input">
-  <div class="section" ng-repeat="(sectionName, section) in sections">
-    <h4>{{section.name}}<br>
-      <small><i>{{section.description}}</i></small>
-    </h4>
+  <div class="background">
```

Comment:
The default coding style puts no indent after body, since it doesn't really serve a purpose, so when I auto-reformat it unindents this base div. (this coincidentally fixes charges being too long of a line)